.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -1,5 +1,5 @@
 ---
-name: "Feature Request"
+name: "\U0001F680 Feature Request"
```

Comment:
`\U0001F680`
What's this?

Reply:
:bug:

Reply:
:rocket: We can remove this emoji if you think it is not necessary